addons/website/static/src/snippets/s_features_grid/000.scss

```diff
@@ -1,13 +1,11 @@
 
 .s_features_grid {
     .s_features_grid_content {
-        overflow: hidden;
         p {
             margin-bottom: 0;
```

Comment:
Let's remove this rule, like for `s_features` snippet (see https://github.com/odoo/odoo/pull/163577/files#r1679061023)